charts/sn-platform/templates/broker/broker-cluster.yaml

```diff
@@ -280,6 +289,14 @@ spec:
     custom:
       PULSAR_PREFIX_additionalServlets: "license-plugin-servlet"
       PULSAR_PREFIX_additionalServletDirectory: "./brokerAdditionalServlet"
+      {{- if .Values.tls.broker.enabled }}
+      PULSAR_PREFIX_tlsCertificateFilePath: "/pulsar/certs/broker/tls.crt"
+      PULSAR_PREFIX_tlsKeyFilePath: "/pulsar/certs/broker/tls.key"
+      PULSAR_PREFIX_tlsAllowInsecureConnection: "true"
+      {{- if not .Values.certs.public_issuer.enabled }}
```

Comment:
The condition seems to be a bit weird

Reply:
The presence or absence of this condition has the same effect, I think that the public certificate does not need to specify an additional CA certificate. But if it is confusing, removing it should be better.